docs/supportedsites.rst

```diff
@@ -129,6 +129,7 @@ Twitter              https://twitter.com/                |twitter-C|
 VSCO                 https://vsco.co/                    Collections, individual Images, User Profiles
 Wallhaven            https://wallhaven.cc/               individual Images, Search Results                  Optional (`API Key <configuration.rst#extractorwallhavenapi-key>`__)
 Warosu               https://warosu.org/                 Threads
+Weasyl               https://weasyl.com/                 Submissions, Folders, Journals
```

Comment:
You're not supposed to add this here manually, but rather add it via scripts/supportedsites.py and have it generate it.